pkg/registry/core/serviceaccount/storage/token.go

```diff
@@ -191,6 +210,9 @@ func (r *TokenREST) Create(ctx context.Context, name string, obj runtime.Object,
 		Token:               tokdata,
 		ExpirationTimestamp: metav1.Time{Time: nowTime.Add(time.Duration(out.Spec.ExpirationSeconds) * time.Second)},
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.ServiceAccountTokenJTI) {
+		audit.AddAuditAnnotation(ctx, serviceaccount.JTIKey, sc.ID)
```

Comment:
do we want to unconditionally record the JTI if it is set in a token? If there _were_ some weird bug/side effect of this, I'd be concerned if someone is totally unable to disable this behaviour given it is alpha 👀

Reply:
It is probably best to completely disable the feature when the gate is not enabled.